sources/libcore/systemInformation.cpp

```diff
@@ -67,6 +99,9 @@ namespace cage
 			memcpy(CPUBrandString + (16 * i), CPUInfo, sizeof(CPUInfo));
 		}
 		return CPUBrandString;
+#elif defined CAGE_SYSTEM_LINUX
+		holder<processHandle> prg = newProcess(string("cat /proc/cpuinfo | grep -m 1 'model name' | cut -d: -f2"));
```

Comment:
please change it to -f2- so that we do not lose part of the name if it contains colon

Reply:
Added in new version of code